components/automate-ui/src/app/pages/integrations/list/integrations-list.component.html

```diff
@@ -67,3 +67,19 @@ <h2 class="display4 strong">Cloud Platform Providers</h2>
     </main>
   </div>
 </div>
+
+<chef-modal
```

Comment:
I actually used this, and then reverted it.  With the current wording, using `<app-delete-object-modal>` does not have enough flexibility, unfortunately.

Reply:
Should be a straightforward change to get what you need by (a) adding a `custom` flag to the `<app-delete-object-modal>`, then display `<ng-content>` if `custom` is true, otherwise display the standard verbiage. Let's chat Monday if you need more details.